cmd/queue/main_test.go

```diff
@@ -443,6 +443,38 @@ func TestQueueTraceSpans(t *testing.T) {
 	}
 }
 
+func BenchmarkProxyHandler(b *testing.B) {
+	var httpHandler http.HandlerFunc = func(w http.ResponseWriter, r *http.Request) {}
+	server := httptest.NewServer(httpHandler)
+	serverURL, _ := url.Parse(server.URL)
+	defer server.Close()
+	proxy := httputil.NewSingleHostReverseProxy(serverURL)
+	params := queue.BreakerParams{QueueDepth: 10, MaxConcurrency: 10, InitialCapacity: 10}
```

Comment:
We should probably create a permutation of this test that uses an infinity breaker (for containerConcurrency == 0), to be able to compare performance.